hack/make/gccgo

```diff
@@ -8,6 +8,9 @@ BINARY_FULLNAME="$BINARY_NAME$BINARY_EXTENSION"
 
 source "$(dirname "$BASH_SOURCE")/.go-autogen"
 
+if [[ "${BUILDFLAGS[@]}" =~ "netgo" ]]; then
+	EXTLDFLAGS_STATIC_DOCKER="$EXTLDFLAGS_STATIC_DOCKER -lnetgo"
```

Comment:
This could just be `EXTLDFLAGS_STATIC_DOCKER+=' -lnetgo'` to simplify somewhat further.